### DISCUSSION TITLE
Thanks. Will it support loading offline map packages in the future？

### DISCUSSION THREAD

**Discussion by buzaiqizhong** [Serg-Norseman/GEDKeeper#607] [General]
It's great to have location tagging through a map. 
But Chinese users have some difficulty using Google Maps and Bing Maps. 
It would be perfect if it could support offline maps. 
Thanks to the author...

**Comment by Serg-Norseman** [comment]
Thank you for your kind review!

To implement loading of any offline map packages, they need to be - such packages. And since the program is free open source - the packages must be free and freely available. Unlikely.

Currently, all mapping services work through API keys with a limited or paid number of requests. The restrictions are growing and becoming insurmountable.
